accelerator/core.c

```diff
@@ -6,10 +6,10 @@
  * "LICENSE" at the root of this distribution.
  */
 
-#include "common_core.h"
+#include "core.h"
 #include <sys/time.h>
 
-#define CC_LOGGER "common_core"
+#define CC_LOGGER "core"
```

Comment:
May you rename `CC_LOGGER` to `CORE_LOGGER`? Since `CC_LOGGER` is the abbreviation of "Common Core LOGGER", thanks!.